validator/package-lock.json

```diff
@@ -1,8 +1,398 @@
 {
```

Comment:
Is this needed to be checked in?

Reply:
This was updated when I pulled from [`ampprojectl:main`](https://github.com/ampproject/amphtml) into my branch [`rtCamp:add/amp-audio-bento`](https://github.com/rtCamp/amphtml/tree/add/amp-audio-bento).
If I have missed something, please let me know any necessary steps needed from my end.

Commit Reference: https://github.com/rtCamp/amphtml/commit/c55f1c17d8969d19f0962e678396bb264127d68b

Reply:
I see it is reverted in the next commit. Ty